hypertrace-graphql-metric-schema/src/main/java/org/hypertrace/graphql/metric/request/DefaultMetricAggregationRequestBuilder.java

```diff
@@ -157,11 +167,20 @@ private Integer percentileSize(Map<String, Object> arguments) {
     AttributeModel attribute;
     AttributeModelMetricAggregationType aggregation;
     List<Object> arguments;
+    List<SelectedField> selectedFields;
 
     @Override
     public String alias() {
       return String.format(
           "%s_%s_%s", this.aggregation.name(), this.attribute.id(), this.arguments);
     }
+
+    @Override
+    public Baseline baseline() {
+      if (selectedFields.size() > 0) {
+        return new Baseline();
+      }
+      return null;
```

Comment:
never null - this likely goes away based on my comment on the baseline class - seems like a boolean.